components/server/Cargo.toml

```diff
@@ -30,6 +30,7 @@ backup = { path = "../backup", default-features = false }
 cdc = { path = "../cdc", default-features = false }
 chrono = "0.4"
 tempfile = "3.0"
+causal_ts = { path = "../causal_ts", default-features = false }
```

Comment:
Why set default-features explicitly?

Reply:
En...I see that almost all our components is set as `default-features = false`, so I think it would be conventional to do this.
`causal_ts` do not have default features, so the setting here has no effect.

Reply:
They are written with `default-features = false` for a reason. Just don't make code verbose.